src/lazy/binary/raw/value.rs

```diff
@@ -258,7 +258,7 @@ impl<'a, 'top> EncodedBinaryValueData_1_0<'a, 'top> {
     /// value's body (that is: the content of the value that follows its opcode and length).
     pub fn body_range(&self) -> Range<usize> {
         let encoded = &self.value.encoded_value;
-        let body_offset = encoded.header_length();
+        let body_offset = encoded.annotations_header_length as usize + encoded.header_length();
```

Comment:
🗺️ This fixes a bug that the `ion-cli` [has to work around](https://github.com/amazon-ion/ion-cli/blob/059a95ce8422121f591a6062db8df6c929f00f53/src/bin/ion/commands/inspect.rs#L928-L936).